casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -31,6 +31,8 @@ import coop.rchain.casper.util.rholang.{InterpreterUtil, RuntimeManager}
 import coop.rchain.casper.util.ProtoUtil
 import coop.rchain.casper.util.rholang.InterpreterUtil
 
+import monix.execution.Scheduler.Implicits.global
```

Comment:
I don't think any file in the `main` code should import the global scheduler. I think it is better to put an implicit parameter on the specific method that needs a scheduler.

Reply:
Alright